extensions/notion/src/components/forms/CreatePageForm.tsx

```diff
@@ -66,7 +67,7 @@ export function CreatePageForm({ mutate, defaults }: CreatePageFormProps) {
             primaryAction: {
               title: "Open Page",
               shortcut: { modifiers: ["cmd"], key: "o" },
-              onAction: () => handleOnOpenPage(page, useRecentPages().setRecentPage),
+              onAction: () => handleOnOpenPage(page, setRecentPage),
```

Comment:
The original error was that it broke the Rule of Hooks. Originally, `useRecentPages` (a hook) was called inside a function. Fix was just to call `useRecentPages` inside the component